main.js

```diff
@@ -0,0 +1,19 @@
+const mobileMenu = document.querySelector('.menu-bar');
+
+function closeMenu() {
+  mobileMenu.style.left = '-110%';
+}
```

Comment:
- [ ] In this case, if the user clicks or taps on any options, the mobile menu should be closed. However, when the user clicks on the option About, it is not disappearing. Please try to avoid this issue as well.

![review](https://user-images.githubusercontent.com/69011963/142687747-7a1901eb-0441-4766-81ef-418618e98d09.gif)